docs/en/advanced-concepts/settings.md

```diff
@@ -72,6 +72,7 @@ Settings are implemented via the following API endpoints:
 | `scrollAcknowledgmentTimeout` | Same as: [setScrollAcknowledgmentTimeout](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html#setScrollAcknowledgmentTimeout(long)). If a negative value is given, it would set to default (200 milliseconds). Handled by [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) in Android API 18 and above. | e.g. `300` |
 | `serverPort` | Controls the server port. Integer values between `1024` and `65535` are available. Defaults to `6790`. | `1024` to `65535` |
 | `shutdownOnPowerDisconnect` | Shutdown the server through the broadcast receiver on [ACTION_POWER_DISCONNECTED](https://developer.android.com/reference/android/content/Intent.html#ACTION_POWER_DISCONNECTED). Defaults to `true`. | `false` or `true` |
+| `simpleBoundsCalculation` | If enabled then the `bounds` attribute will be calculated using a less accurate but simpler method that may improve performance. Defaults to `false`. | `false` or `true` |
```

Comment:
it also makes sense to mention the minimum Appium version where this setting is available

Reply:
ok, cool; how do I find out what the next version of appium will be?  Will it be 1.17.2 or 1.18.0?

Reply:
It will be `1.18.0` for now.
(If this do not include in https://github.com/appium/appium/pull/14484 , I'll update the version)

Reply:
Ok, I added a note about appium 1.18.0.